superset-frontend/src/reports/actions/reports.js

```diff
@@ -164,7 +164,7 @@ export function deleteActiveReport(report) {
         dispatch(addDangerToast(t('Your report could not be deleted')));
       })
       .finally(() => {
-        dispatch(deleteReport(report.id));
+        dispatch(deleteReport(report));
```

Comment:
were we going to fetch an empty report instead of delete from state manually?

Reply:
I played around with this, and it doesn't work very well with the new way that we structured the fetch action to get rid of the racing condition for hydrating a dashboard. 

I was working through the logic for making the setReport action also consider a deleteAction, and then realized that I was just writing a delete action, so wrote it that way instead.